### ISSUE TITLE
[Doc] Opening Dialogs

### ISSUE THREAD

**Issue by mysteryx93** [AvaloniaUI/Documentation#258]
There is a page (3 pages) in the documentation about [opening dialogs here](https://docs.avaloniaui.net/tutorials/music-store-app/opening-a-dialog). It's a very complicated approach.

I just published [HanumanInstitute.MvvmDialogs](https://github.com/mysteryx93/HanumanInstitute.MvvmDialogs) which makes this kind of operation super simple. I think the documentation should be updated to include this much simpler approach. With at least a link; the library's usage is well-documented already.

**Comment by MikeCodesDotNET** [comment]
I agree that the current docs for dialogs need some improvement. Dialogs are one of those tricky things to do, especially when trying to stick with MVVM. Our current docs only demonstrate how to do it with ReactiveUI, which has a steep learning curve for those unfamiliar with Reactive development. 

The docs show how to use RxUI for the dialogs included in the project templates, which means it should ‘just work’ for noobs. There is probably a broader conversation about whether we should include RxUI at all, but that’s probably best for another day. 

## Possible solution

One option might be to make the dialogs page include links to the various supported libraries with a brief description. The actual docs for the libraries probably shouldn’t be included, as I’d be concerned that the information wasn’t kept up to date.

**Comment by mysteryx93** [comment]
As of right now, there is no other MVVM solution, but there might be in the future. Having out-of-date doc can indeed be an issue.

One option is to create a page to describe the topic and problem, and short description+link to various solutions.

The current page isn't actually a dialog page; but an explanation for the specific demo code.
